src/Intervention/Image/Image.php

```diff
@@ -1687,9 +1687,9 @@ public function parseColor($value)
             $rgbaPattern = '/^rgba ?\(([0-9]{1,3}), ?([0-9]{1,3}), ?([0-9]{1,3}), ?([0-9.]{1,4})\)$/i';
 
             if (preg_match($hexPattern, $value, $matches)) {
-                $r = strlen($matches[1]) == '1' ? '0x'.$matches[1].$matches[1] : '0x'.$matches[1];
-                $g = strlen($matches[2]) == '1' ? '0x'.$matches[2].$matches[2] : '0x'.$matches[2];
-                $b = strlen($matches[3]) == '1' ? '0x'.$matches[3].$matches[3] : '0x'.$matches[3];
+                $r = strlen($matches[1]) == '1' ? hexdec($matches[1].$matches[1]) : hexdec($matches[1]);
+                $g = strlen($matches[2]) == '1' ? hexdec($matches[1].$matches[2]) : hexdec($matches[2]);
+                $b = strlen($matches[3]) == '1' ? hexdec($matches[1].$matches[3]) : hexdec($matches[3]);
```

Comment:
Looks like there is a copy-paste mistake. Should be `$matches[2].$matches[2]` and `$matches[3].$matches[3]`.